packages/grafana-data/src/transformations/transformers/ids.ts

```diff
@@ -6,6 +6,7 @@ export enum DataTransformerID {
   order = 'order', // order fields based on user configuration
   organize = 'organize', // order, rename and filter based on user configuration
   rename = 'rename', // rename field based on user configuration
+  calculateField = 'calculateField', // Run a reducer on the row
```

Comment:
Another name idea would be `reduceRow`?

Reply:
:) this was my original name -- but thinking that we want to distinguish it more from the other "reduce" transformer that has very different behavior.

Reply:
ok, i got you!